package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "fantasy-scoring",
   "version": "1.0.0",
-  "description": "",
+  "description": "## Instructions In the initial commit to this project, you have been provided with a basic project setup as well as a set of tests which currently fail. Your task is to create a `calculateScore` function which will properly calculate the fantasy score for all known positions, returning 0 when the position is not one of the ones laid out below.",
```

Comment:
Looks like you ran `npm init`. Remember `npm init` is for when we don't already have a `package.json` file. This project already had one, all you needed to do was run `npm install`. When you run `npm install` with no other parameters it will install all packages listed in `package.json`.